src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -3039,6 +3040,115 @@ def test_sql_managed_instance_mgmt(self):
                  expect_failure=True)
 
 
+class SqlManagedInstancePoolScenarioTest(ScenarioTest):
+    @record_only()
+    def test_sql_instance_pool(self):
+
+        instance_pool_name_1 = "clitestipbwikkrkamao"
+        instance_pool_name_2 = "clitestipbwikkrkamao2"
+        license_type = 'LicenseIncluded'
+        location = 'northcentralus'
+        v_cores = 8
+        edition = 'GeneralPurpose'
+        family = 'Gen5'
+        resource_group = 'billingPools'
+        vnet_name = 'vnet-billingPool1'
+        subnet_name = 'InstancePool'
+        subnet = self.cmd('network vnet subnet show -g {} --vnet-name {} -n {}'.format(resource_group, vnet_name, subnet_name)).get_output_in_json()['id']
+
+        # test create sql managed_instance
+        self.cmd(
```

Comment:
Please add test for --on-wait and wait command.

Reply:
Done